test/app-tap/gh-5040-inter-mode-isatty.test.lua

```diff
@@ -35,7 +35,7 @@ local testcases = {
         cmd_args = '',
         stdin = '/dev/null',
         interactive = false,
-        empty_output = true
+        empty_output = false
```

Comment:
I suspect that `echo 'print(42)' | tarantool` is unintentionally affected. When `isatty(0) == 0` the script should be read from stdin. I guess this behavior is not supposed to be changed.

Reply:
Discussed in group. Here is the decision:
1. `echo 'print(42)' | tarantool` yields usage message
2. `echo 'print(42)' | tarantool -` yields `42` (i.e. executes Lua script)
3. The change should be mentioned in a ChangeLog entry.

Reply:
Shouldn't we fix the test case by changing the `cmd_args` to `' -'` (and keep `empty_output = true`) to don't change the meaning of the test case?